src/utils/hooks/useLoadReview.tsx

```diff
@@ -27,41 +34,39 @@ const useLoadReview = ({ reviewId, serialNumber }: UseLoadReviewProps) => {
     serialNumber,
   })
 
-  const { triggerProcessing, error: triggerError } = useTriggerProcessing({
-    reviewId,
-    // triggerType: 'reviewTrigger',
-  })
-
   const { error: responsesError, responsesByCode, elementsState } = useGetResponsesAndElementState({
     serialNumber,
     isApplicationLoaded,
   })
 
-  const { data, error: apolloError, loading: apolloLoading } = useGetReviewQuery({
+  const { data, error, loading } = useGetReviewQuery({
     variables: {
       reviewId,
     },
-    skip: triggerProcessing || !isApplicationLoaded || !elementsState,
+    skip: !isApplicationLoaded || !elementsState,
+    fetchPolicy: 'network-only',
   })
 
-  useEffect(() => {
-    if (applicationError) {
-      const error = applicationError as ApolloError
-      setError(error.message)
-      return
-    }
-    if (application) setApplicationName(application.name)
-  }, [applicationError, application])
+  const { triggerProcessing, error: triggerError } = useTriggerProcessing({
+    checkTrigger,
+    reviewId,
+    // triggerType: 'reviewTrigger',
+  })
+
+  const { data: statusData, error: statusError, loading: statusLoading } = useGetReviewStatusQuery({
```

Comment:
Shouldn't this one also pass in `triggerProcessing` like the Application one does? (triggerProcessing is currently not being used for anything)

Reply:
Yes, that was just an attempt to use the trigger for review, but hasn't finished - because there are still #204 issue to add the action to create the first `review_status_history` when the review is created. I though it would be good to do that when using this. But good point, I'll add the `triggerProcessing` check here.